app/views/artists/show.html.erb

```diff
@@ -15,8 +33,36 @@
   <%# <h5 class="card-title"><%= @artist.name </h5> %>
     <%# <p class="card-text"><%= @artist.description </p> %>
       </div>
+>>>>>>> master
   </div>
 
+<<<<<<< HEAD
+<div class="artist-video-wrapper">
+  <h3>Artist Videos</h3>
+ 
+  
+<div class="row row-cols-4">
+
+<% if @artist.first_video_id.nil? %>
+<div>Empty.</div>
+<% else %>
+<div class="plyr__video-embed" id="player">
+  <iframe
+    src="<%= @artist.youtube_embed_url %>?origin=https://plyr.io&amp;iv_load_policy=3&amp;modestbranding=1&amp;playsinline=1&amp;showinfo=0&amp;rel=0&amp;enablejsapi=1"
+    allowfullscreen
+    allowtransparency
+    allow="autoplay"
+  ></iframe>
+</div>
+<% end %>
+</div>
+
+</div>
+
+<div class="";>
+<h3>Current Booked Events</h3>
+  <% if @artist.events.any? %>
+=======
   <div class="artist-video-wrapper">
       <h3>Artist Videos</h3>
     <div class="row row-cols-4">
```

Comment:
there's a merge conflict here you should address

Reply:
cleaned up